modules/runtime/pages/bonita-docker-installation.adoc

```diff
@@ -194,22 +194,6 @@ services:
     restart: on-failure:2
     depends_on:
       - db
-    entrypoint:
-      - bash
-      - -c
-      - |
-        set -e
-        echo 'Waiting for PostgreSQL to be available'
-        maxTries=10
-        while [ "$$maxTries" -gt 0 ] && [ $$(echo 'QUIT' | nc -w 1 "$$DB_HOST" 5432; echo "$$?") -gt 0 ]; do
```

Comment:
That's why we now use `nc` instead of PSQL.
So I **do not understand why you made this PR**